plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -542,7 +542,7 @@ public void createDatabase(HiveIdentity identity, Database database)
     }
 
     @Override
-    public void dropDatabase(HiveIdentity identity, String databaseName)
+    public void dropDatabase(HiveIdentity identity, String databaseName, boolean deleteData)
```

Comment:
add `// TODO respect deleteData`

Reply:
Updating this, I noticed I'll probably have to make a similar change in `InMemoryThriftHiveMetastore`.